nx-X11/lib/X11/XKeysymDB

```diff
@@ -353,7 +353,7 @@ XF86Suspend		:1008FFA7
 XF86Hibernate		:1008FFA8
 XF86TouchpadToggle	:1008FFA9
 XF86TouchpadOn		:1008FFB0
-XF86TouchpadOff	:1008FFB1
+XF86TouchpadOff		:1008FFB1
```

Comment:
Yes, it make sense. With a tab size of 8 the previous version looked ugly, now it look good.